source/docs/software/actuators/pneumatics.rst

```diff
@@ -3,12 +3,68 @@
 Operating pneumatic cylinders
 =============================
 
+Using the FRC Control System to control Pneumatics
+------------------------------------------------------------------------------------
+
+.. note:: The Pneumatics Control Module (PCM) is a CAN-based device that provides complete control over the compressor and up to 8 solenoids per module. The PCM is integrated into WPILib through a series of classes that make it simple to use. The closed loop control of the Compressor and Pressure switch is handled by the PCM hardware and the Solenoids are handled by the upgraded Solenoid class that now controls the solenoid channels on the PCM. An additional PCM module can be used where the modules corresponding solenoids are differentiated by the module number in the constructors of the Solenoid and Compressor classes.
+
+.. image:: images/pcm.jpg
+    :width: 400
+
+The Pneumatics Control Module from CTR Electronics is responsible
+for regulating the robot's pressure using a pressure switch and a
+compressor and switching solenoids on and off. The PCM communicates
+with the RoboRIO over CAN. For more information, see `FRC Control System Hardware Overview`
+
```

Comment:
It'd be good to add the section about pcm module numbers from [screensteps](https://wpilib.screenstepslive.com/s/currentCS/m/java/l/599708-operating-pneumatic-cylinders-solenoids#pcm_module_numbers)
```suggestion

PCM Module Numbers
------------------

PCM Modules are identified by their Node ID. The default Node ID for PCMs is 0. If using a single PCM on the bus it is recommended to leave it at the default Node ID. 

```

Reply:
I agree, however I'm not sure Node ID is the correct word. In the CAN documents it's called a CAN ID or Device ID, and I'm not sure what (if anything) is gained from using the word Node instead of Device, for the sake of consistency.

Reply:
I agree. I was just copying what screensteps had, and Device id is a better wording